src/v2/es-utils.js

```diff
@@ -191,19 +222,19 @@ class ESClass {
     let reason = response.reason || 'n/a';
     if (response.hasOwnProperty('result')) {
       result = response.result.toUpperCase();
-    } else if (response.hasOwnProperty('total')) {
-      if (response.hasOwnProperty('created') && response.created > 0)
-        result = 'CREATED (' + response.created + ' items)';
-      if (response.hasOwnProperty('deleted') && response.deleted > 0)
-        result = 'DELETED (' + response.deleted + ' items)';
-      if (response.hasOwnProperty('updated') && response.updated > 0)
-        result = 'UPDATED (' + response.updated + ' items)';
-    } else if (response.hasOwnProperty('acknowledged')) {
+    }
+    if (response.hasOwnProperty('total')) {
+      const totalCount = response.created + response.deleted + response.updated;
+      result = 'CHANGED (' + totalCount + ' items)';
```

Comment:
Why do you use template literals everywhere except this line?

Reply:
I have removed this code in favor of simplified output (i.e. only report errors)